pennylane/_device.py

```diff
@@ -264,6 +264,10 @@ def shots(self, shots):
                 "Shots must be a single non-negative integer or a sequence of non-negative integers."
             )
 
+    @property
+    def shot_vector(self):
```

Comment:
```suggestion
    def shot_vector(self):
    """list[.ShotTuple[int, int]]: Returns the shot vector, a sparse
    representation of the shot sequence used by the device
    when evaluating QNodes.

    **Example**

    >>> dev = qml.device("default.qubit", wires=2, shots=[3, 1, 2, 2, 2, 2, 6, 1, 1, 5, 12, 10, 10])
    >>> dev.shots
    57
    >>> dev.shot_vector
    [ShotTuple(shots=3, copies=1),
     ShotTuple(shots=1, copies=1),
     ShotTuple(shots=2, copies=4),
     ShotTuple(shots=6, copies=1),
     ShotTuple(shots=1, copies=2),
     ShotTuple(shots=5, copies=1),
     ShotTuple(shots=12, copies=1),
     ShotTuple(shots=10, copies=2)]

    The sparse representation of the shot
    sequence is returned, where tuples indicate the number of times a shot
    integer is repeated.
    """
```

Reply:
I had this docstring lying around in one of my local files. I haven't tested it, but I recommend running the example code just to make sure that's what it outputs 😆